packages/babel-core/test/fixtures/transformation/source-maps/input-source-map-multiple-output-sources/source-map.json

```diff
@@ -1,6 +1,6 @@
 {
   "version": 3,
-  "mappings": "AAAC,KCAD;;AACA,SAASA,GAAT,CAAaC,GAAb,EAAwB;EACpB,MAAM,IAAIC,KAAJ,CAAU,cAAV,CAAN;AACH",
+  "mappings": "AAAC,KAAG;;ACCJ,SAASA,GAAG,CAACC,GAAW;EACpB,MAAM,IAAIC,KAAK,CAAC,cAAc,CAAC;AACnC",
```

Comment:
Changes are fine. Same reasoning with the `;` pointing to the same file as the expression.